addons/compact-menu-bar/addon.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Compact editor menu bar",
```

Comment:
If the only thing this addon does is hide the username, we might want to use a different name as making the whole menu bar compact sounds more involved than just hiding the username. In the Customizable navigation bar addon, that setting is called "compact user dropdown".

Reply:
Done.